detail.py

```diff
@@ -0,0 +1,34 @@
+from django.conf import settings
+from django_mako_plus import view_function, jscontext
+from datetime import datetime, timezone
+from catalog import models as cmod
+import math
```

Comment:
I'm not sure the necessity of importing math for this file.

Reply:
yeah, I just copied code from a separate .py file 😅